relay-general/src/store/transactions/processor.rs

```diff
@@ -1882,6 +1890,26 @@ mod tests {
         "/testing/open-19-close/1",
         "/testing/*/1"
     );
+    transaction_name_test!(
+        test_transaction_name_normalize_url_encode_1,
+        "/%2Ftest%2Fopen%20and%20help%2F1%0A",
+        "/%2Ftest%2Fopen%20and%20help%2F1%0A"
+    );
+    transaction_name_test!(
+        test_transaction_name_normalize_url_encode_2,
+        "/this/1234/%E2%9C%85/foo/bar/098123908213",
+        "/this/*/%E2%9C%85/foo/bar/*"
+    );
+    transaction_name_test!(
+        test_transaction_name_normalize_url_encode_3,
+        "/foo/hello%20world-4711/",
+        "/foo/hello%20world-4711/"
+    );
+    transaction_name_test!(
+        test_transaction_name_normalize_url_encode_4,
+        "/foo/hello%20world-0xdeadbeef/",
+        "/foo/hello%20world-0xdeadbeef/"
+    );
```

Comment:
Another edge case that would be nice to test is an encoded slash (`%2F`).

Reply:
+1